### PR TITLE
Select new typefaces

### LINKED ISSUE #26: Select new font faces
We need a dyslexia font and another one that doesn't require updating `font-family` for bold text.

The user will be able to select among at least two fonts, possibly more.

### PR DESCRIPTION
**Pull request by saabi**
Closes #26

### CODE REVIEW

File: src/node_modules/app/components/AccessibilityMenu.svelte [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	import {get} from 'svelte/store';
+	import * as _ from 'lamb';
+	import {fontFamily, fontScaling, fontVariationSettings} from 'app/stores/fontScaling';
+
+
+	const fonts = [
+		'Archivo',
+		'Open Dyslexia'
+	];
+	const fontSizes = [0.75, 1.0, 1.25];
```
Comment (mindrones) [inline src/node_modules/app/components/AccessibilityMenu.svelte]: I'd rename to `fontScales`

File: src/node_modules/app/components/AccessibilityMenu.svelte [hunk 2]
```diff
@@ -0,0 +1,74 @@
+<script>
+	import {onMount} from 'svelte';
+	import {get} from 'svelte/store';
+	import * as _ from 'lamb';
+	import {fontFamily, fontScaling, fontVariationSettings} from 'app/stores/fontScaling';
+
+
```
Comment (mindrones) [inline src/node_modules/app/components/AccessibilityMenu.svelte:6]: leftover blank line

File: src/node_modules/app/stores/fontScaling.js [hunk 3]
```diff
@@ -1,3 +1,7 @@
 import {writable} from "svelte/store";
```
Comment (mindrones) [inline src/node_modules/app/stores/fontScaling.js]: I'd rename this file to just `font.js`

File: src/routes/_layout.svelte [hunk 4]
```diff
@@ -18,8 +19,12 @@
...
-	$: rootStyle 
+	$: rootStyle
+		&& (rootStyle.fontFamily = $fontFamily);
+	$: rootStyle
 		&& (rootStyle.fontSize = `calc(${defaultFontSize} * ${$fontScaling})`);
+	$: rootStyle
+		&& (rootStyle.fontVariationSettings = $fontVariationSettings);
 </script>
 
 <ScreenGauge bands={[60, 82, 100, 120]} devMode={dev} />
```
Comment (mindrones) [inline src/routes/_layout.svelte:30]: Since we're here I'd use default `bands` (that is, I'd not use `bands`)

File: static/global.css [hunk 5]
```diff
@@ -1,21 +1,62 @@
...
+	src: url('/font/OpenSans-Semibold.ttf') format('truetype');
+}
+@font-face {
+    font-family: 'clfallback';
+    src: url('/font/OpenDyslexic3-Regular.otf') format('opentype')
+}
+*/
+@font-face {
+    font-family: 'Open Dyslexia';
+    src: url('/font/OpenDyslexic-Regular.otf') format('opentype');
```
Comment (mindrones) [inline static/global.css]: I'd move `/font/OpenDyslexic*` to `/font/OpenDyslexic/*`

File: static/global.css [hunk 6]
```diff
@@ -1,21 +1,62 @@
...
+}
+@font-face {
+    font-family: 'Open Dyslexia';
+    src: url('/font/OpenDyslexic-Bold-Italic.otf') format('opentype');
+    font-weight: 700;
+    font-style: italic
+}
+@font-face {
+    font-family: 'Archivo';
+    src: url('/font/Archivo-VariableFont_wdth,wght.ttf') format('truetype');
```
Comment (mindrones) [inline static/global.css]: I'd move `/font/Archivo*` and `/font/OFL.txt` to `/font/Archivo/*`

File: static/global.css [hunk 7]
```diff
@@ -1,21 +1,62 @@
...
-  font-family: 'Open Sans Regular';
-  src: url('/font/OpenSans-Regular.ttf') format('truetype');
+	font-family: 'Open Sans Regular';
+	src: url('/font/OpenSans-Regular.ttf') format('truetype');
 }
 @font-face {
-  font-family: 'Open Sans Semibold';
-  src: url('/font/OpenSans-Semibold.ttf') format('truetype');
+	font-family: 'Open Sans Semibold';
+	src: url('/font/OpenSans-Semibold.ttf') format('truetype');
```
Comment (mindrones) [inline static/global.css]: I'd rename `/font/unused/` `/font/unused-OpenSans/*` and I'd move `/font/OpenSans*` to `/font/unused-OpenSans/*`

**Review by mindrones — CHANGES_REQUESTED** [review]
first pass

File: static/global.css [hunk 8]
```diff
@@ -1,21 +1,62 @@
...
+    src: url('/font/Archivo-Italic-VariableFont_wdth,wght.ttf') format('truetype');
+    font-weight: 400;
+    font-style: italic
 }
 
 * {
 	box-sizing: border-box;
 	margin: 0;
 	padding: 0;
-	font-family: 'Open Sans Light', sans-serif;
```
Comment (mindrones) [inline static/global.css:18]: Just a note that as we discussed we need to restore some fallback, at least `sans-serif`.

File: src/routes/_layout.svelte (hunk 4 above)
Comment (saabi) [reply src/routes/_layout.svelte:30]: This should probably be dealt with in #16

**Review by mindrones — APPROVED** [review]
Great work, good to merge 👌

**Pull request merged**